### ISSUE TITLE
Android Auto cannot be whitelisted manually

### ISSUE THREAD

**Issue by SoNic1967** [ggsava/block-this#48]
Wireless Android Auto cannot be found in the list of applications, to be whitelisted. 
Can it be hardcoded? There is no real reason to block the Wireless Android Auto Wi-Fi connection.

**Comment by onitreb** [comment]
Yes, this would be great. Is there any known workaround besides disabling block-this?
